tools/install.sh

```diff
@@ -238,10 +206,21 @@ echo "CentOS distribution detected."
 # for CentOS, the log directory is called httpd
 logdirectory=/var/log/httpd
 while true; do
-    read -p "Would you like to automatically create/install apache config files? (In development for CentOS 6.5) [yn] " yn
+    read -p "Would you like to automatically set up your apache configuration files? (In development for CentOS) [yn] " yn
```

Comment:
> (In development for CentOS)

Is this true? Does this work or not? It would be good to have an issue specifically for this.

Reply:
Not sure if someone is developing it but Ubuntu and CentOS have different basic config using different file structure. So change applied by the script might be overridden by default configuration due to order of appearance.

Reply:
OK well it sounds like further testing should be done to make sure that Apache can be automatically configured by the install script for CentOS. I'll make an issue for it.